lldb/source/Plugins/DynamicLoader/FreeBSD-Kernel/DynamicLoaderFreeBSDKernel.cpp

```diff
@@ -153,9 +153,10 @@ addr_t DynamicLoaderFreeBSDKernel::FindKernelAtLoadAddress(
 }
 
 // Read ELF header from memry and return
+template <typename Elf_Ehdr>
```

Comment:
Exactly why does this change? I don't see a need for it in this PR.

Reply:
The need is in here:
```c++
llvm::ELF::Elf64_Ehdr header;
    if (!ReadELFHeader(process, addr, header)) {
      *read_error = true;
      return UUID();
    }
```

In this case the header is Elf64 instead of Elf32. So I try to use generic to handle different version of Elf